src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -222,6 +222,7 @@ export const TermsAuthEntry = React.createClass({
         stageParams: PropTypes.object.isRequired,
         errorText: PropTypes.string,
         busy: PropTypes.bool,
+        hideContinue: PropTypes.bool,
```

Comment:
Usual plea for positive options rather than negative ones, ie. `showContinue` with default `true` rather than the double-negavtive `hideContinue: false`.